### ISSUE TITLE
Chef 13 support

### ISSUE THREAD

**Issue by runningman84** [hw-cookbooks/graphite#238]
please support Chef version 13:
```
    - /var/chef/cache/cookbooks/graphite/resources/carbon_aggregator.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property name of resource graphite_carbon_cache. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/carbon_cache.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property name of resource graphite_carbon_conf_accumulator. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/carbon_conf_accumulator.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property name of resource graphite_carbon_relay. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/carbon_relay.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property name of resource graphite_service. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/service.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property prefix of resource graphite_storage. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/storage.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property name of resource graphite_storage_conf_accumulator. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/storage_conf_accumulator.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property name of resource graphite_storage_schema. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/storage_schema.rb:23:in `class_from_file'
  Cannot specify both default and name_property together on property path of resource graphite_web_config. Only one (name_property) will be obeyed. In Chef 13, this will become an error. at 1 location:
    - /var/chef/cache/cookbooks/graphite/resources/web_config.rb:22:in `class_from_file'
  version nil currently does not overwrite the value of version. This will change in Chef 13, and the value will be set to nil instead. Please change your code to explicitly accept nil using "property :version, [MyType, nil]", or stop setting this value to nil. at 3 locations:
```

**Comment by spuder** [comment]
+1

**Comment by spuder** [comment]
Fixed in pull request #245

**Comment by tas50** [comment]
Fixed by #245
